### PR TITLE
Pcvl 500 doc

### PR DESCRIPTION
**Pull request by melvinmathe**
Little changes in legacy.rst, usage.rst and tools.rst. Some images were replaced.

### CODE REVIEW

File: docs/source/usage.rst [hunk 1]
```diff
@@ -8,130 +8,19 @@ To use perceval, you can directly install the pip wheel in a virtual environment
...
 Alternatively, if you are interested in contributing to the project - you can clone the project from github:
 
 .. code-block:: bash
 
-   (.venv) $ git clone https://github.com/quandela/Perceval
-   (.venv) $ cd Perceval
-   (.venv) $ python setup.py install # or develop for developers
+   (venv) $ git clone https://github.com/quandela/Perceval
+   (venv) $ cd Perceval
+   (venv) $ python setup.py install # or develop for developers
```
Comment (MarionQuandela) [inline docs/source/usage.rst:19]: Can you also change `python setup.py install` into `pip install .` here and everywhere in this repo

File: docs/source/tools.rst [hunk 2]
```diff
@@ -92,6 +92,19 @@ Then, add ``pcvl.pdisplay()`` of your circuit.
...
+.. tip::
+
+    The outcome of this last command will depend on your environment.
+
+    .. list-table::
+       :header-rows: 1
+       :width: 100%
+
+       * - Text Console
+         - Jupyter Notebook
```
Comment (ericbrts) [inline docs/source/tools.rst:104]: Is it normal that you removed the IDE case in this table? I guess it's still something we want to show

File: docs/source/usage.rst [hunk 3]
```diff
@@ -8,130 +8,19 @@ To use perceval, you can directly install the pip wheel in a virtual environment
...
-| |0,1> |  1/2  |  1/2  |   0   |   0   |   0   |
-| |1,0> |  1/2  |  1/2  |   0   |   0   |   0   |
-| |1,1> |   0   |   0   |  1/2  |   0   |  1/2  |
-+-------+-------+-------+-------+-------+-------+
-
-Congratulations, you have achieved this first tutorial! You can now continue with the documentation through
-:ref:`Circuits`, :ref:`Computing Backends`, :ref:`States`, :ref:`Polarization` or you will go through more advanced
-examples in the `Examples` section starting with :ref:`Getting started with Perceval`.
+At this point you can directly use Perceval.
+If you want to learn how to build Linear Optical circuits, manipulate vector states or run quantum algorithms remotely,
```
Comment (ericbrts) [inline docs/source/usage.rst:25]: state vectors (not vector states)

**Review by ericbrts — CHANGES_REQUESTED** [review]

File: README.md [hunk 4]
```diff
@@ -52,7 +52,7 @@ or simply from GitHub:
 
 ```bash
 git clone https://github.com/quandela/Perceval
-python setup.py install # [or 'develop' for developers]
```
Comment (ericbrts) [inline README.md:55]: I just have one last thing: the comment `# [or 'develop' for developers]` should turn into : 
`# or pip install -e . for developers`

Could you please add this mention back at both places you updated the install command?
Then the PR is good!

Comment (melvinmathe) [reply README.md:55]: done !

**Review by ericbrts — APPROVED** [review]

**Pull request merged**